packages/matchbox/src/components/Slider/Slider.js

```diff
@@ -1,18 +1,65 @@
-/* eslint max-lines: ["error", 260] */
+/* eslint max-lines: ["error", 303] */
 import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
+import styled from 'styled-components';
+import { margin, compose } from 'styled-system';
+import { createPropTypes } from '@styled-system/prop-types';
 import { getRectFor, lerp, useWindowSize } from '../../helpers/geometry';
 import { noop, isNotTouchEvent } from '../../helpers/event';
 import { onKey, onKeys } from '../../helpers/keyEvents';
 import { roundToPlaces, clamp } from '../../helpers/math';
-import styles from './Slider.module.scss';
+import { slider, rail, track, tick, tickLabel, handle, handleShadow } from './styles';
+
+const system = compose(margin);
+
+export const StyledSlider = styled('div')`
+  ${slider}
+  ${system}
+`;
+
+const StyledRail = styled('div')`
+  ${rail}
+`;
+
+const StyledTrack = styled('div')`
+  ${track}
+`;
+
+const StyledTick = styled('div')`
+  ${tick}
+`;
+
+const StyledTickLabel = styled('div')`
+  ${tickLabel}
+`;
+
+const StyledHandle = styled('div')`
+  ${handle}
+`;
+
+const StyledHandleShadow = styled('div')`
+  ${handleShadow}
+`;
 
 function Slider(props) {
-  const { defaultValue, disabled, id, max, min, onBlur, onFocus, onChange, precision, ticks, value } = props;
+  const {
+    defaultValue,
+    disabled,
+    id,
+    max,
+    min,
+    onBlur,
+    onFocus,
+    onChange,
+    precision,
+    ticks,
+    value,
+  } = props;
```

Comment:
Will need to pass through system props for margin, I think and apply them to StyledSlider or another wrapper around anything